wiki/Abandoned_wasteland/id.md

```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - forum
+  - wastelanded
+---
+
+# Abandoned wasteland
```

Comment:
```suggestion
# Gurun yang terbengkalai
```

Reply:
do we still need to use english, i think it's hard to understand for this one

Reply:
we really should not